python_on_whales/components/container/cli_wrapper.py

```diff
@@ -666,6 +670,17 @@ def create(
 
         The arguments are the same as [`docker.run`](#run).
         """
+        if isinstance(command, str):
+            error_message = textwrap.dedent(
+                f"""\
+                When calling docker.create(), the second argument ('command') should be a sequence of strings.
+                Here are some examples:
+                 docker.create('ubuntu', ['ls'])
+                 docker.create('ubuntu', ['cat', '/some/file.txt'])
+                You can try docker.create('{image}', {shlex.split(command)}, ...).
```

Comment:
Thank you for this error message. This is so, so awesome. Would it be possible in another PR to add a unit tests for this? Something that checks that `docker.create('{image}', {shlex.split(command)}` appears correctly in the error message.